### PR TITLE
Typo: String strip method docstring 'remove' -> 'removed'.

### PR DESCRIPTION
**Pull request by mbanerjeepalmer**
Seems it should be what the docs say: https://docs.python.org/3/library/stdtypes.html?highlight=strip#str.strip

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Review by aeros — APPROVED** [review]
Thanks for the PR @mbanerjeepalmer and welcome! Good catch.

Since this is correcting a docstring, I'll categorize it as being a documentation change. The typo correction looks good to me, but because this file is using [Argument Clinic](https://docs.python.org/3/howto/clinic.html), we'll have to run ``Tools/clinic/clinic.py`` on it to update the checksum. That's why Travis is failing. 

Regarding the CLA, if the status is not updated within 24 hours of signing it, you may have to refresh the status on GitHub manually: https://check-python-cla.herokuapp.com. 

We'll have to wait for a core developer to review the changes before merging it (and for the CLA to be signed), but let me know if you have any questions in the meantime. If you haven't already, I'd recommend checking out the ["Lifecycle of a Pull Request" section of the devguide](https://devguide.python.org/pullrequest/).

**Comment by mbanerjeepalmer** [comment]
Thanks for explaining @aeros. I signed the CLA 24 hours ago but the checker hasn't updated yet. I'll look again tomorrow to ensure I've done everything correctly.

**Comment by aeros167** [comment]
> I signed the CLA 24 hours ago but the checker hasn't updated yet.

Did you create an account on bugs.python.org as well? Once the "Contributor Form Received" status is updated on there, it should update on GitHub soon after.

**Comment by mbanerjeepalmer** [comment]
Looks like the CLA issue is resolved now.

Regarding Argument Clinic, I'm afraid I'm not entirely clear what I need to do next. Could you point me to where I can find instructions? It's not clear which part of the docs relate to what I'm trying to do here.

**Comment by aeros167** [comment]
> Regarding Argument Clinic, I'm afraid I'm not entirely clear what I need to do next. Could you point me to where I can find instructions? It's not clear which part of the docs relate to what I'm trying to do here.

The main documentation for Argument Clinic is on https://docs.python.org/3/howto/clinic.html. 

I'm not particularly experienced with using Argument Clinic (as my PRs have not involved changes to C files). As far as I'm aware, all you should have to do is run clinic.py on the C file to update the checksum``./Tools/clinic/clinic.py Objects/clinic/unicodeobject.c.h`` for Unix-based systems, or ``py Tools/clinic/clinic.py Objects/clinic/unicodeobject.c.h`` for Windows. You may need to provide ``Tools/clinic/clinic.py`` with execute permissions. 

This should be done locally within the branch being used for the PR (in your case, "master", which is the default branch), commit the changes, and then push the changes directly to the PR branch. If you're not overly familiar with using Git, I can try to help to walk you through that process.

Note: It's generally not advisable to open a PR from your master, as it makes it difficult to have multiple open PRs in the same repository. Before opening a new PR, I usually will update master, create a new branch, make changes within the branch, and then open the PR using the branch. It should be okay for this PR, but I just wanted to let you know in case you weren't aware. (:

Git Bootcamp: https://devguide.python.org/gitbootcamp/. The guide was mainly written in the context of contributing to CPython, but it's an excellent resource for Git usage in general.

File: Objects/clinic/unicodeobject.c.h
Comment (zware) [inline Objects/clinic/unicodeobject.c.h:585]: This is a generated file.  Can you instead make the correction in `Objects/unicodeobject.c` and then run `make clinic`?

**Review by zware — CHANGES_REQUESTED** [review]

File: Objects/clinic/unicodeobject.c.h
Comment (aeros) [reply Objects/clinic/unicodeobject.c.h:585]: @zware:
>  Can you instead make the correction in Objects/unicodeobject.c and then run make clinic?

Ah, I wasn't aware that Argument Clinic could not be ran against the header file directly, but that would make sense. I've personally never had to use it before, so I've only briefly read over the documentation to try to help others.

Also, I was not aware of the existence of ``make clinic``, I thought the only option was to explicitly use ``./Tools/clinic/clinic.py <path>``.  It's not mentioned within the devguide as far as I can tell: https://devguide.python.org/search/?q=make+clinic&check_keywords=yes&area=default.

Should we perhaps mention it somewhere? Using ``make clinic`` is far more straightforward than using clinic.py on each of the C files that need it.

Comment (zware) [reply Objects/clinic/unicodeobject.c.h:585]: Hmm, it looks like we also don't mention `make regen-all` anywhere in the devguide, which is probably the better target to remember since it includes `make clinic`.  On the other hand neither is available on Windows, so we should probably also mention the clinic script directly.

**Comment by csabella** [comment]
@mbanerjeepalmer, please address the last comment by @zware.  Thank you!

**Comment by csabella** [comment]
This is a minor change, but seems to have been abandoned.  I'm going to close this pull request.  It can be reopened if work will continue on it or a new pull request can be opened to replace it.  Thank you!

**Pull request closed without merging**
